contracts/FastBreakV1.cdc

```diff
@@ -13,47 +13,52 @@
 
 import NonFungibleToken from 0xNFTADDRESS
 import TopShot from 0xTOPSHOTADDRESS
+import MetadataViews from 0xMETADATAVIEWSADDRESS
 
 /// Game & Oracle Contract for Fast Break V1
 ///
-pub contract FastBreakV1: NonFungibleToken {
+access(all) contract FastBreakV1: NonFungibleToken {
+
+    access(all) entitlement Play
+    access(all) entitlement Create
+    access(all) entitlement Update
 
     /// Contract events
     ///
-    pub event ContractInitialized()
+    access(all) event ContractInitialized()
 
-    pub event Withdraw(id: UInt64, from: Address?)
+    access(all) event Withdraw(id: UInt64, from: Address?)
 
-    pub event Deposit(id: UInt64, to: Address?)
+    access(all) event Deposit(id: UInt64, to: Address?)
```

Comment:
I'd recommend removing these because the default `NonFungibleToken` events are much better